io/mod.ts

```diff
@@ -3,6 +3,5 @@ export * from "./buffer.ts";
 export * from "./bufio.ts";
 export * from "./ioutil.ts";
 export * from "./readers.ts";
-export * from "./streams.ts";
```

Comment:
Moving these to `std/streams` looks reasonable to me. But how about keeping alias to `std/streasm/conversion.ts` with `@deprecated` annotations? That makes the transitions easier for the users.

Reply:
> Moving these to `std/streams` looks reasonable to me. But how about keeping alias to `std/streasm/conversion.ts` with `@deprecated` annotations? That makes the transitions easier for the users.

how would an alias have specific `@deprecated` annotation?